system/HTTP/Files/UploadedFile.php

```diff
@@ -373,5 +373,26 @@ public function isValid(): bool
 		return is_uploaded_file($this->path) && $this->error === UPLOAD_ERR_OK;
 	}
 
+	/**
+     * Save the uploaded file to a new location.
+     *
+     * By default.upload files are saved in writable/uploads directory. the YYYYMMDD folder
+     * and random file name will be created.
+     *
+     * @param string $folderName the folder name to writable/uploads directory.
+     * @param string $fileName   the name to rename the file to.
+     * @return string file full path
+     */
+    public function store($folderName = null, $fileName = null) : string
```

Comment:
How is this any different than the move command that you're using in the middle of it?

Reply:
Saving a file is much simpler and requires only one line of code to save the file.